scripts/kdb.fish

```diff
@@ -241,8 +241,8 @@ function __fish_kdb_print_resolver_plugins -d 'Print a list of available resolve
 end
 
 function __fish_kdb_print_storage_plugins -d 'Print a list of available storage plugins'
-    set -l formats constants crypto_botan crypto_gcrypt crypto_openssl desktop dpkg dump fcrypt hosts line ini json ni passwd regexstore
-    set -l formats $formats simpleini simplespeclang tcl xmltool uname
+    set -l formats constants crypto_botan crypto_gcrypt crypto_openssl desktop dpkg dump fcrypt hosts line ini json mini ni passwd
+    set -l formats $formats regexstore simpleini simplespeclang tcl xmltool uname
```

Comment:
That sounds like a good idea, although I would prefer we just add the possibility to list only storage plugins to `kdb list`. We could for example add an optional argument (`storage`, `resolver`, …) to the command. This way it would still be possible to use `kdb.fish` directly.